cmd/up.go

```diff
@@ -716,7 +716,7 @@ func (up *UpContext) cleanCommand() {
 		in,
 		&out,
 		os.Stderr,
-		[]string{"sh", "-c", "(((cp /var/okteto/bin/* /usr/local/bin); (ps -ef | grep -v -E '/var/okteto/bin/syncthing|/var/okteto/bin/remote|PPID' | awk '{print $2}' | xargs -r kill -9)) >/dev/null 2>&1); cat /proc/sys/fs/inotify/max_user_watches"},
+		[]string{"sh", "-c", "(((cp /var/okteto/bin/* /usr/local/bin); (/var/okteto/bin/clean)) >/dev/null 2>&1); cat /proc/sys/fs/inotify/max_user_watches"},
```

Comment:
I would move the full line to `clean` to simplify this code.
Also, if `clean` is not present, we should execute the previous command